hc/api/tests/test_list_checks.py

```diff
@@ -33,15 +33,38 @@ def get(self):
 
     def test_it_works(self):
         r = self.get()
+        self.assertEqual(200,r.status_code)
         ### Assert the response status code
 
         doc = r.json()
         self.assertTrue("checks" in doc)
 
         checks = {check["name"]: check for check in doc["checks"]}
         ### Assert the expected length of checks
```

Comment:
I notice that you have not removed the prompts. In PEP8 we are supposed to keep comments up to date with the code. Please act on this by removing this comment or explaining what the code does at the moment.

Reply:
Updated check commit [update comments](https://github.com/andela/hc-june-bunnies/pull/5/commits/13e2098dd20b6dac5139f6e1132abc9000223261)